Podcast/FollowsEndpointRequests.swift

```diff
@@ -54,31 +55,15 @@ class FetchUserFollowsByIDRequest: EndpointRequest {
     
 }
 
-class FollowUserEndpointRequest: EndpointRequest {
-    
+class ModifyFollowEndpointRequest: EndpointRequest {
     // ID to follow
     var userID: String
     
-    init(userID: String) {
-        self.userID = userID
-        super.init()
-        
-        path = "/followings/\(userID)/"
-        httpMethod = .post
-    }
-}
-
-class UnfollowUserEndpointRequest: EndpointRequest {
-    
-    // ID to unfollow
-    var userID: String
-    
-    init(userID: String) {
+    init(userID: String, method: HTTPMethod) {
```

Comment:
not sure if we should be passing the HTTP method here directly because it's not 100% understandable from a reader's perspective (ie. "post" a follow doesn't really make sense), maybe an enum called `FollowAction` or something like that could be useful here?

Reply:
+1

Reply:
I agree, I originally had an enum `Action` that had `create` or `delete` and an `httpMethod` var. I'll change it back!